Sources/ParseSwift/Types/ParseError.swift

```diff
@@ -406,3 +406,90 @@ extension ParseError: CustomStringConvertible {
         debugDescription
     }
 }
+
+// MARK: Compare Errors
+extension Error {
+    /**
+     Validates if the given ParseError Code is equal to the error.
+     
+    **Example use case:**
+    ~~~
+    if error.equalsTo(.objectNotFound)  {
+        //Do stuff
+    }
+    ~~~
+     - parameter errorCode: The ParseError code you want to compare to
+     */
+    public func equalsTo(_ errorCode: ParseError.Code) -> Bool {
+        guard let error = self as? ParseError else {
+            return false
+        }
+        return error.code == errorCode
+    }
+
+    /**
+     Returns an optional ParseError if the given ParseError Code is equal to the error.
+     
+    **Example use case:**
+    ~~~
+    if let parseError = error.equalsTo(.objectNotFound)  {
+        print(parseError.description)
+    }
+    ~~~
+     - parameter errorCode: The ParseError code you want to compare to
+     
+     - returns: An optional ParseError
+     */
+    public func equalsTo(_ errorCode: ParseError.Code) -> ParseError? {
+        guard let error = self as? ParseError else {
+            return nil
+        }
+        if error.code == errorCode {
+            return error
+        } else {
+            return nil
+        }
+    }
+
+    /**
+     Validates if the given ParseError Codes contains the error.
+     
+    **Example use case:**
+    ~~~
+    if error.contains([.objectNotFound, .invalidQuery])  {
+        //Do stuff
+    }
+    ~~~
+     - parameter errorCode: The ParseError codes as **Array** you want to compare to
+     */
+    public func contains(_ errorCode: [ParseError.Code]) -> Bool {
+        guard let error = self as? ParseError else {
+            return false
+        }
+        return errorCode.contains(error.code)
+    }
+
+    /**
+     Returns an optional ParseError if the given ParseError Codes contains the error.
+     
+    **Example use case:**
+    ~~~
+    if let parseError = error.contains([.objectNotFound, .invalidQuery])  {
+        print(parseError.description)
+    }
+    ~~~
+     - parameter errorCode: The ParseError codes as **Array** you want to compare to
+     
+     - returns: An optional ParseError
+     */
+    public func contains(_ errorCode: [ParseError.Code]) -> ParseError? {
```

Comment:
You should also add the variadic counterparts for your methods that take an array as input. You can copy/paste/modify the test cases for them. The one for this one is below:

```swift
/**
     Returns an optional ParseError if the given ParseError Codes contains the error.
     
    **Example use case:**
    ~~~
    if let parseError = error.containedInParseError(.objectNotFound, .invalidQuery)  {
        print(parseError.description)
    }
    ~~~
     - parameter errorCode: A variadic amount of zero or more of `ParseError` codes to compare to.
     
     - returns: An optional `ParseError`
     */
    public func containedIn(_ errorCode: ParseError.Code...) -> ParseError? {
        self.containedIn(errorCode)
    }
```

Take notes of the adjustments I made to your comments. The comments you add should look more like this. For example, you should modify the comments on all of your methods to look similar and remove words like "you" when you can